common/src/JetCorrections.cxx

```diff
@@ -428,6 +428,8 @@ JetLeptonCleaner::~JetLeptonCleaner(){}
 JetLeptonCleaner_by_KEYmatching::JetLeptonCleaner_by_KEYmatching(uhh2::Context& ctx, const std::vector<std::string> & filenames, const std::string& jet_label){
 
     h_jets_ = ctx.get_handle<std::vector<Jet>>(jet_label);
+    h_topjets_ = ctx.get_handle<std::vector<TopJet>>(jet_label);
+    label_ = jet_label;
```

Comment:
is this "label_" class member necessary? as far as i can see, it's not used anywhere